docs/tasks/administer-cluster/dns-custom-nameservers.md

```diff
@@ -58,36 +98,37 @@ details about the configuration option format.
 
 {% capture discussion %}
 
-## Understanding name resolution in Kubernetes
-
-DNS policies can be set on a per-pod basis. Currently Kubernetes supports two pod-specific DNS policies: “Default” and “ClusterFirst”. These policies are specified with the `dnsPolicy` flag.
+## Impacts on Pods
```

Comment:
I think this should be a sub-section under `Configure stub-domain and upstream DNS servers`?

Reply:
make senses, moving it down one level